mir-module/src/main/resources/config/mir/messages_de.properties

```diff
@@ -100,513 +100,518 @@ metadata.versionInfo.startLabel = (Gesamte Versionsgeschichte)
-mir.validation.sdnb                    = Bitte w\u00E4hlen Sie eine passende SDNB-Kategorie aus.
-mir.validation.title                   = Bitte geben Sie einen Titel ein.
-mir.validation.url                     = Bitte korrigieren Sie den Link (beginnend mit http://, https:// oder ftp://).
-mir.validation.urn                     = Bitte korrigieren Sie die URN (urn:nbn:...).
-mir.volume                             = Band, Jahrgang:
+mir.abstract                            = Abstract
+mir.abstract.aslink                     = als Link:
+mir.abstract.editor                     = (Hrsg.)
+mir.abstract.hideGroups                 = alles einklappen
+mir.abstract.in                         = Abstract in:
+mir.abstract.link                       = Link zum Abstract:
+mir.abstract.ortext                     = oder Flie\u00DFtext:
+mir.abstract.readless                   = \u00AB einklappen
+mir.abstract.readmore                   = weiterlesen \u00BB
+mir.abstract.showGroups                 = alles ausklappen
+mir.abstract.text                       = Abstract, Beschreibung:
+mir.abstract.text.2                     = Abstract, Beschreibung
+mir.abstract.translated                 = Abstract-\u00DCbersetzung
+mir.access                              = Zugriffsberechtigung
+mir.accesskey                           = Zugriffsschl\u00FCssel
+mir.accesskey.add                       = Zugriffsschl\u00FCssel hinzuf\u00FCgen
+mir.accesskey.delete                    = Zugriffsschl\u00FCssel l\u00F6schen
+mir.accesskey.edit                      = Zugriffsschl\u00FCssel bearbeiten
+mir.accesskey.generate                  = Erzeugen
+mir.accesskey.matches                   = Die eingegebenen Zugriffsschl\u00FCssel d\u00FCrfen nicht identisch sein!
+mir.accesskey.read                      = Leseschl\u00FCssel
+mir.accesskey.read.required             = Bitte geben Sie einen Leseschl\u00FCssel ein!
+mir.accesskey.required                  = Bitte geben Sie einen Zugriffsschl\u00FCssel ein.
+mir.accesskey.setOnUser                 = Zugriffsschl\u00FCssel eintragen
+mir.accesskey.write                     = Schreibschl\u00FCssel
+mir.accesskey.write.required            = Bitte geben Sie einen Schreibschl\u00FCssel ein!
+mir.actions                             = Aktionen
+mir.actions.noaccess                    = Um das Dokument zu bearbeiten melden sie sich bitte an.
+mir.actions.norights                    = Sie besitzen nicht die ben\u00F6tigten Rechte um das Dokument zu bearbeiten.
+mir.admineditor                         = Bearbeiten im Admin-Editor
+mir.affiliation                         = Zugeh\u00F6rigkeit
+mir.breadcrumb.home                     = Start
+mir.cartographics.coordinates           = Geographische Koordinaten:
+mir.cartographics.coordinates.box       = Box
+mir.cartographics.coordinates.geoType   = Geometrie Typ
+mir.cartographics.coordinates.none      = Keines
```

Comment:
please fix:
 - Geometrie Typ -> Geometrie-Typ
 - none / Keines -> please choose / bitte wählen ("none" can be misunderstood)